README.md

```diff
@@ -26,7 +33,60 @@ struct Person {
 }
 
 fn main() {
-    let conn = Connection::connect("postgres://postgres@localhost", SslMode::None).unwrap();
+    let uri = "postgres://postgres@localhost";
+    let conn = match Connection::connect(uri, SslMode::None) {
+        Ok(conn) => conn,
+        Err(e) => {
+            println!("Connection error: {}", e);
+            return;
+        }
+    };
+    conn.execute("CREATE TABLE person (
+                    id              SERIAL PRIMARY KEY,
+                    name            VARCHAR NOT NULL,
+                    data            BYTEA
+                  )", &[]).unwrap();
+    let me = Person {
+        id: 0,
+        name: "Steven".to_string(),
+        data: None,
+    };
+    conn.execute("INSERT INTO person (name, data) VALUES ($1, $2)",
+                 &[&me.name, &me.data]).unwrap();
+    for row in &conn.query("SELECT id, name, data FROM person", &[]).unwrap() {
+        let person = Person {
+            id: row.get(0),
+            name: row.get(1),
+            data: row.get(2),
+        };
+        println!("Found person {}", person.name);
+    }
+}
+```
+
+Same example as above but with SSL
+```rust
```

Comment:
I'll reduce the example

On Aug 14, 2016 4:43 PM, "Steven Fackler" <notifications@github.com> wrote:

> In README.md
> <https://github.com/sfackler/rust-postgres/pull/194#discussion_r74711061>:
>
> > +    };
> > +    conn.execute("INSERT INTO person (name, data) VALUES ($1, $2)",
> > +                 &[&me.name, &me.data]).unwrap();
> > +    for row in &conn.query("SELECT id, name, data FROM person", &[]).unwrap() {
> > +        let person = Person {
> > +            id: row.get(0),
> > +            name: row.get(1),
> > +            data: row.get(2),
> > +        };
> > +        println!("Found person {}", person.name);
> > +    }
> > +}
> > +```
> > +
> > +Same example as above but with SSL
> > +```rust
>
> It seems like we don't need to duplicate the entire example - just the
> context setup and connection line should be enough to show people what to
> do.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sfackler/rust-postgres/pull/194/files/d5bfc161a799f5044c6dd93df8a394d43ffcb828#r74711061>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABMFFqLl6j-c1mPzKQ5M28RgNCH_6Mc8ks5qf6gXgaJpZM4JeYP7>
> .
>